context-monitoring/src/main/java/de/atb/context/persistence/monitoring/MonitoringDataRepository.java

```diff
@@ -57,7 +50,7 @@
  * @author scholze
  * @version $LastChangedRevision: 143 $
  */
-public final class MonitoringDataRepository<Type extends IMonitoringDataModel<?, ?>> extends RepositorySDB<Type> implements
+public final class MonitoringDataRepository<Type extends IMonitoringDataModel<?, ?>> extends RepositoryTDB<Type> implements
```

Comment:
Could `instance` be final?

Reply:
I do not get the point

Reply:
`private static volatile MonitoringDataRepository<? extends IMonitoringDataModel<?, ?>> instance;`

`instance` only seems to be assigned in the `static` block. So could be final.